tests/cases/fourslash/jsdocReturnsTag.ts

```diff
@@ -17,7 +17,6 @@ verify.signatureHelp({
     text: "find<T>(l: T[], x: T): T",
     docComment: "Find an item",
     tags: [
-        // TODO: GH#24130

```

Comment:
Doesn't look like #24130 is fixed here -- text still contains "\n ".

Reply:
Added the comment back in, with a pointer to this PR.